code/modules/mining/mineral_effect.dm

```diff
@@ -1,6 +1,6 @@
 /obj/effect/mineral
 	name = "mineral vein"
-	icon = 'icons/obj/mining.dmi'
+	icon = 'icons/obj/mining_vr.dmi' //VOREStation Edit

```

Comment:
I am a bit concerned about this considering the potential of it breaking things coming in from upstream.

Reply:
If they add more ores, you mean? That's pretty unlikely! We've got other things like this too, like we had computers for quite some time, among other things.

I'll add an error iconstate to make it obvious if something breaks, and to let miners still see the 'new' ore until fixed. :p